### PR TITLE
Remove missions created by automation

### PR DESCRIPTION
**Pull request by swimsoft**
- Remove missions created by automation
- Fix offenses

### CODE REVIEW

File: db/data_migrations/20210121113148_remove_missions_created_by_automation.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class RemoveMissionsCreatedByAutomation < ActiveRecord::DataMigration
+  def up
+    if Rails.env.production?
+      name = Mission.arel_table[:name]
```
Comment (Alexey1100) [inline db/data_migrations/20210121113148_remove_missions_created_by_automation.rb:4]: Let's add a spec here :)

E.g: `spec/data_migrations/fill_wallet_for_account_token_record_spec.rb`

**Review by Alexey1100 — DISMISSED** [review]
Specs

File: db/data_migrations/20210121113148_remove_missions_created_by_automation.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+      Mission.where(name.matches('%Automation%')).destroy_all
```
Comment (matjacdev515) [inline db/data_migrations/20210121113148_remove_missions_created_by_automation.rb:5]: Think `Mission.where("name LIKE '%Automation%'").destroy_all` will be more developer-friendly)

**Review by Riddlerrr — APPROVED** [review]

File: db/data_migrations/20210121113148_remove_missions_created_by_automation.rb (hunk 2 above)
Comment (aquabu) [reply db/data_migrations/20210121113148_remove_missions_created_by_automation.rb:5]: @swimsoft I'm going to merge without resolving this comment from @matjacdev515 to keep the code moving. But I think @matjacdev515. Can you add the change to a future commit?

**Pull request merged**